src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -0,0 +1,249 @@
+import _ from 'underscore';
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import {ScrollView} from 'react-native-gesture-handler';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../../../ONYXKEYS';
+import ROUTES from '../../../ROUTES';
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import compose from '../../../libs/compose';
+import KeyboardAvoidingView from '../../../components/KeyboardAvoidingView/index';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import MenuItem from '../../../components/MenuItem';
+import CONST from '../../../CONST';
+import variables from '../../../styles/variables';
+import ExpensifyText from '../../../components/ExpensifyText';
+import ExpensifyButton from '../../../components/ExpensifyButton';
+import FixedFooter from '../../../components/FixedFooter';
+import CurrentWalletBalance from '../../../components/CurrentWalletBalance';
+import * as paymentPropTypes from './paymentPropTypes';
+import * as PaymentMethods from '../../../libs/actions/PaymentMethods';
+import * as PaymentUtils from '../../../libs/PaymentUtils';
+import getBankIcon from '../../../components/Icon/BankIcons';
+
+const propTypes = {
+    /** User's wallet information */
+    userWallet: paymentPropTypes.userWalletPropTypes,
+
+    /** Array of bank account objects */
+    bankAccountList: paymentPropTypes.bankAccountListPropTypes,
+
+    /** Array of card objects */
+    cardList: paymentPropTypes.cardListPropTypes,
+
+    /** Wallet balance transfer props */
+    walletTransfer: paymentPropTypes.walletTransferPropTypes,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    userWallet: {
+        walletLinkedAccount: {},
+    },
+    bankAccountList: [],
+    cardList: [],
+    walletTransfer: {},
+};
+const Fee = 0.30;
```

Comment:
Feels like this should this come from the backend.

Reply:
Yeah, but there is nothing that gives us this info from backend so I had to hardcode it.